sql/moz-fx-data-shared-prod/telemetry_derived/cohort_daily_statistics_v1/query.sql

```diff
@@ -40,10 +40,11 @@ cohorts_in_range AS (
   FROM
     telemetry_derived.rolling_cohorts_v1
   WHERE
-    cohort_date > DATE_SUB(
+    cohort_date >= DATE_SUB( -- Note this is a pretty big scan... Look here for problems
       @activity_date,
       INTERVAL 180 DAY
-    ) -- Note this is a pretty big scan... Look here for problems
+    )
+    AND cohort_date < @activity_date -- No need to get activity for the cohort date - everyone will be retained
```

Comment:
Would a `BETWEEN` be a bit easier to read here?

Reply:
Yup, good call